authors/settings.py

```diff
@@ -24,8 +24,6 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
-
```

Comment:
@Emmanuelaaron l would like to know why you removed line 27 in your settings.py that is `ALLOWED_HOSTS`
Am l missing out on anything

Reply:
The line was removed because it had no purpose.